driver-core/src/main/com/mongodb/internal/operation/EstimatedDocumentCountOperation.java

```diff
@@ -105,6 +118,9 @@ private CommandCreator getCommandCreator(final SessionContext sessionContext) {
             BsonDocument document = new BsonDocument("count", new BsonString(namespace.getCollectionName()));
             appendReadConcernToCommand(sessionContext, connectionDescription.getMaxWireVersion(), document);
             putIfNotZero(document, "maxTimeMS", maxTimeMS);
+            if (comment != null) {
```

Comment:
Nit: Could also use the helper: `putIfNotNull(document, "comment", comment);`

Reply:
I copied this from another operation, so at least it's consistent.